pkg/blobaccess/standard/access.go

```diff
@@ -0,0 +1,77 @@
+package standard
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/mandelsoft/filepath/pkg/filepath"
+	"github.com/opencontainers/go-digest"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess/bpi"
+	"github.com/open-component-model/ocm/pkg/iotools"
+	mimetypes "github.com/open-component-model/ocm/pkg/mime"
+)
+
+type _nopCloser = iotools.NopCloser
+
+////////////////////////////////////////////////////////////////////////////////
+
+type bytesAccess struct {
+	_nopCloser
+	data   []byte
+	origin string
+}
+
+func DataAccessForData(data []byte, origin ...string) bpi.DataSource {
+	path := ""
+	if len(origin) > 0 {
+		path = filepath.Join(origin...)
+	}
+	return &bytesAccess{data: data, origin: path}
+}
+
+func DataAccessForString(data string, origin ...string) bpi.DataSource {
```

Comment:
```suggestion
func DataSourceForString(data string, origin ...string) bpi.DataSource {
```

Reply:
see above

Reply:
`dataAccess.ForXXX(...)`?

Reply:
`dataAccess` is no package. It is contained in the `blobaccess` package. Therefore, there are two flavors:
- `blobaccess.ForXXX(...)`
- `blobaccess.DataAccessForXXX(...)` (optionally available)